operator/roles/kiali-deploy/defaults/main.yml

```diff
@@ -8,86 +8,103 @@
 # If new groups are added to these defaults, you must remember to add the merge code to vars/main.yml.
 
 kiali_defaults:
-  installation_tag: ""
-  istio_namespace: ""
+  installation_tag: ''
```

Comment:
why are all the double-quotes changing to single quotes? These have no bearing on the LDAP PR.

Reply:
VS code is converting automatically when I save

Reply:
Any way changed it back the previous form